QsCompiler.sln

```diff
@@ -55,6 +55,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "QirGeneration", "src\QsComp
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LlvmBindings", "src\QsCompiler\LlvmBindings\LlvmBindings.csproj", "{36FB68CC-533A-40D4-A2E9-597A35C6912C}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E70908F4-CE50-4CDA-A5DE-0C02554FBBC6}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QsCompiler", "QsCompiler", "{8088F458-4201-46A4-882A-047A35A8601D}"
+EndProject
```

Comment:
This shows up a little strangely in solution view. I'm not sure what the best organization is. There could be a folder for C# generation, but moving the C# generation tests to the Tests folder seems weird. Maybe we could get rid of Tests and in general have the test project adjacent to the project it's testing?

![image](https://user-images.githubusercontent.com/33814365/122993674-7054f500-d35c-11eb-8bb8-1ade33346f0d.png)

Reply:
Ah, right. I've run into this before, because I use `dotnet sln add` to handle adding things to the solution file, which follows the folder structure instead of the previous patterns that get created when someone uses the Visual Studio UI. Since I don't have Visual Studio installed, I'm going to try editing the file to get the new projects to show up in the other style.

Reply:
Ok, I think I've got it back into the existing form. I've left the Test folder as-is; I'll leave it up to those who use VS regularly to decide what structure makes sense to that file. My two cents: we should be using the dotnet cli to manage solution files, since that's what anyone working cross-platform would need to use, even if that ends up creating a folder structure in the solution explorer pane for those who use VS.